recipe/meta.yaml

```diff
@@ -1,20 +1,25 @@
+{% set name = "fiona" %}
 {% set version = "1.9.5" %}
 
 package:
   name: fiona
   version: {{ version }}
 
 source:
-  url: https://github.com/Toblerity/Fiona/archive/refs/tags/{{ version }}.zip
-  sha256: 31aef8eb0402c7dcd822ffe576fccbb7bfbe29a31f3b6805b87770d4b6f266e7
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fiona-{{ version }}.tar.gz
+  sha256: 99e2604332caa7692855c2ae6ed91e1fffdf9b59449aa8032dd18e070e59a2f7
 
 build:
-  number: 1
+  number: 2
   skip: true  # [win and vc<14]
   script_env:
     - GDAL_ENABLE_DEPRECATED_DRIVER_GTM=YES
   entry_points:
     - fio = fiona.fio.main:main_group
+  ignore_run_exports:
+    - python
+    - numpy
+    - setuptools
```

Comment:
To avoid warnings like:
```
 WARNING (fiona): interpreter (Python) package conda-forge::python-3.12.0-hab00c5b_0_cpython in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
 WARNING (fiona): run-exports library package conda-forge::numpy-1.26.2-py312heda63a1_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
 WARNING (fiona): interpreted library (Python) package conda-forge::setuptools-68.2.2-pyhd8ed1ab_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```